README.md

```diff
@@ -624,11 +628,11 @@ Miscellaneous partials:
 
 **[Custom Layou](#templatelayout)t**
 
-If you'd like to provide your own layout, provide a path in `template.layout` (relative to `template.cwd`) in the `settings` object.
+If you'd like to provide your own layout, provide a path to `template.layout`.
```

Comment:
Shouldn't we link to `template.layout` here as well? Like we do in the custom partials section below?